aana/core/models/vad.py

```diff
@@ -74,3 +74,41 @@ def to_whisper_dict(self) -> dict:
 """
 List of VadSegment objects.
 """
+
+
+class SDSegment(BaseModel):
```

Comment:
I agree that having a separate speaker pydantic model type is better. My worry was that it might be too fragmented if a new pydantic model is added for every new model. Probably we can get away with one for all speaker-related tasks.